### PR TITLE
Feat:- wishlist page added

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Add product to wishlist from product list 
send items to cart from wishlist

Please review the PR

### CODE REVIEW

**Review by arulk8 — COMMENTED** [review]
Looks good to go.

File: components/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    display: flex;
+    flex-direction: column;
+
+}
+.pos-img {
+    position: relative;
+}
+.heart-icon {
+    position: absolute;
+    top: 230px;
```
Comment (teja538) [inline components/wishlist/wishlist.css:22]: use relative values like em or %

**Review by teja538 — COMMENTED** [review]
looks good

**Comment by SuchitaGhadge** [comment]
Search bar might be little bigger. it will look good. 
I guess login option should not be visible in navigation on wish list. Otherwise looks good to go.

**Comment by SuchitaGhadge** [comment]
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/anurag-kkumarr/reviewers)

File: components/wishlist/wishlist.css (hunk 1 above)
Comment (anurag-kkumarr) [reply components/wishlist/wishlist.css:22]: > I will use %.

**Comment by anurag-kkumarr** [comment]
> Search bar might be little bigger. it will look good. 
I guess login option should not be visible in navigation on wish list. Otherwise looks good to go.

> Is it so, I didn't knew that. I will look into it.

**Comment by SuchitaGhadge** [comment]
Wishlist and cart page is visible only after a successful login. So you can add a userprofile option instead of login. Just a suggestion

**Pull request merged**